RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -172,78 +175,89 @@ struct SubscriptionDetailsView: View {
                         .frame(width: iconWidth)
                     VStack(alignment: .leading) {
-                        Text(localization.commonLocalizedString(for: .refundStatus))
+                        Text(localization.commonLocalizedString(for: .currentPrice))
                             .font(.caption2)
-                            .foregroundColor(Color(UIColor.secondaryLabel))
-                        Text("\(refundRequestStatusMessage)")
-                            .font(.caption)
+                            .foregroundColor(.secondary)
+                            .textCase(.uppercase)
+                        Text("\(subscriptionInformation.price)")
+                            .font(.body)
+                    }
+                }
+
+                if let nextRenewal =  subscriptionInformation.expirationDateString {
+
+                    let expirationString = subscriptionInformation.active ? (
+                        subscriptionInformation.willRenew ?
+                            localization.commonLocalizedString(for: .nextBillingDate) :
+                            localization.commonLocalizedString(for: .expires)
+                    ) : localization.commonLocalizedString(for: .expired)
+
+                    HStack(alignment: .center) {
+                        Image(systemName: "calendar")
+                            .accessibilityHidden(true)
+                            .frame(width: iconWidth)
+                        VStack(alignment: .leading) {
+                            Text("\(expirationString)")
+                                .font(.caption2)
+                                .foregroundColor(.secondary)
+                                .textCase(.uppercase)
+                            Text("\(String(describing: nextRenewal))")
+                                .font(.body)
+                        }
+                    }
+                }
+
+                if let refundRequestStatusMessage = refundRequestStatusMessage {
+                    HStack(alignment: .center) {
+                        Image(systemName: "arrowshape.turn.up.backward")
+                            .accessibilityHidden(true)
+                            .frame(width: iconWidth)
+                        VStack(alignment: .leading) {
+                            Text(localization.commonLocalizedString(for: .refundStatus))
+                                .font(.caption2)
+                                .foregroundColor(.secondary)
+                                .textCase(.uppercase)
+                            Text("\(refundRequestStatusMessage)")
+                                .font(.body)
+                        }
                     }
                 }
             }
-        }.padding()
-            .padding(.horizontal)
-            .background(Color(UIColor.tertiarySystemBackground))
-            .cornerRadius(20)
-            .shadow(color: Color.black.opacity(0.2), radius: 4)
+
+        }
+        .padding(24.0)
```

Comment:
this is entirely subjective, although the corner radius and shadow details below were as well - not sure if we need to pull this out of here